docs/docs/04-multiple-components.md

```diff
@@ -122,13 +122,15 @@ The situation gets more complicated when the children are shuffled around (as in
     return (
       <ol>
         {results.map(function(result) {
-          return <li key={result.id}>{result.text}</li>;
+          return <ListItemWrapper key={result.id} data={result} />;
```

Comment:
This is what's below in the comparison. I think the point of using an `<li>` here is to make it clear how you would do it with HTML elements before using composite components..